changelog.d/2037.change.rst

```diff
@@ -0,0 +1,4 @@
+A bit "cosmetic" changes though needed for implementing more advanced stuff.
+* Renamed args names params for some static methods and marked them with ''@staticmethod''.
+* Moved ''0'' into ''default_order_value" variable.
+* When an entry point now fails to be loaded, it doesn't disrupt building process.
```

Comment:
I can see value in failing rather than suppressing errors. One should be cautious about suppressing errors, as it might be the case that the failed operation might be critical. The warning is helpful, and given the 'fail on warning' behavior of the setuptools test suite, this warning would prevent the core functionality from being suppressed (at least for the duration of the test suite). I think this change encapsulates the main purpose of this PR and probably addresses the underlying bug (which unfortunately isn't referenced here).

Reply:
Yeah, I have added it as one of the ways to deal with #2030.